docs/interactions/Slash_Commands.md

```diff
@@ -679,12 +678,16 @@ Deletes a global command. Returns `204`.
 
 Fetch all of the guild commands for your application for a specific guild. Returns an array of [ApplicationCommand](#DOCS_INTERACTIONS_SLASH_COMMANDS/applicationcommand) objects.
 
+## Bulk Overwrite Global Application Commands % PUT /applications/{application.id#DOCS_TOPICS_OAUTH2/application-object}/commands
+
+Takes a list of application commands, overwriting existing commands that are registered globally for this application. Updates will be available in all guilds after 1 hour. Returns `200` and a list of [ApplicationCommand](#DOCS_INTERACTIONS_SLASH_COMMANDS/applicationcommand) objects. Commands that do not already exist will count toward your daily application command create limit.
+
 ## Create Guild Application Command % POST /applications/{application.id#DOCS_TOPICS_OAUTH2/application-object}/guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/commands
 
 > danger
 > Creating a command with the same name as an existing command for your application will overwrite the old command.
 
-Create a new guild command. New guild commands will be available in the guild immediately. Returns `201` and an [ApplicationCommand](#DOCS_INTERACTIONS_SLASH_COMMANDS/applicationcommand) object.
+Create a new guild command. New guild commands will be available in the guild immediately. Returns `201` and an [ApplicationCommand](#DOCS_INTERACTIONS_SLASH_COMMANDS/applicationcommand) object.  If the command did not already exist, it will count toward your daily application command create limit.
```

Comment:
To clarify this command create limit is per guild yes? Does this share a limit with global commands?

Reply:
yes and no
well yes, but not really at the same time